internal/artifacts/module_arch.go

```diff
@@ -2,14 +2,12 @@ package artifacts
 
 import (
 	"fmt"
+	"github.com/SAP/cloud-mta/mta"
+	"github.com/pkg/errors"
```

Comment:
These should be with the rest of the github.com dependencies

Reply:
done